datasets/allocine/README.md

```diff
@@ -0,0 +1,17 @@
+---
+language: 
+- BCP-47: fr
```

Comment:
this doesn't have the same scheme as the other card (and won't be compatible with our website filtering out of the box), was this intended?

Reply:
No, I included this by mistake. Sorry about that!